docs/resources/datastore.md

```diff
@@ -148,6 +148,7 @@ Required:
 Optional:
 
 - **credentials** (Block List, Max: 1) Root user credentials (see [below for nested schema](#nestedblock--satori_auth_settings--credentials))
+- **enable_personal_access_token** (Boolean) Enables Satori Personal Access Token authentication. Defaults to `false`.
```

Comment:
I would add an explanation that it also requires to enable it in the level account manually

Reply:
+ add the example to the template of the datastore_resource so it will be shown in the datastore.md resource presented in the Terraform documentation

Reply:
Done X2